elixir/apps/domain/lib/domain/auth/adapters/openid_connect.ex

```diff
@@ -324,6 +325,47 @@ defmodule Domain.Auth.Adapters.OpenIDConnect do
     end
   end
 
+  defp log_claims(provider, tokens) do
+    with {:ok, access_token} <- parse_jwt(tokens["access_token"]),
+         {:ok, id_token} <- parse_jwt(tokens["id_token"]) do
+      Logger.info("Access Token Claims",
+        provider_id: provider.id,
+        access_token: inspect(access_token.fields)
+      )
+
+      Logger.info(inspect(access_token.fields))
+
+      Logger.info("ID Token Claims",
+        provider_id: provider.id,
+        id_token: inspect(id_token.fields)
+      )
+
+      Logger.info(inspect(id_token.fields))
+    else
+      {:error, reason} ->
+        Logger.info("Error parsing JWT",
+          provider_id: provider.id,
+          reason: inspect(reason)
```

Comment:
```suggestion
          reason: reason
```

(nit) I don't think you need inspect here because it's a string, but don't feel strongly.

Reply:
I agree with you and the reason I put this as an inspect is because I wanted to be extra careful here and not have any possibility of crashing the process.  I can take it out if you'd like though.

Reply:
no need since it's temporary